server/server.go

```diff
@@ -196,6 +212,19 @@ func (s *Server) StreamedListObjects(req *openfgapb.StreamedListObjectsRequest,
 	if err != nil {
 		return err
 	}
+
+	model, err := s.datastore.ReadAuthorizationModel(ctx, storeID, modelID)
+	if err != nil {
+		return serverErrors.HandleError("", err)
+	}
+
+	typesys := typesystem.New(model)
+
+	connectObjCmd := &commands.ConnectedObjectsCommand{
+		Datastore:  s.datastore,
+		Typesystem: typesys,
+	}
```

Comment:
why is `ResolveNodeLimit: s.config.ResolveNodeLimit,` not here?

Reply:
Good catch. Resolved!